### ISSUE TITLE
Work in edits/suggestions from `prodesRB.pdf`

### ISSUE THREAD

**Issue by drphilmarshall** [rogerblandford/Music#39]
I'll start, then hand off.

**Comment by drphilmarshall** [comment]
OK, I worked backwards: Stage 2 and 3 and everything after are now RDB-compliant. @LaurencePeanuts can you take care of teh intro and Stage 1 please? Thanks!

**Comment by rogerblandford** [comment]
Looks good.
Please keep Ziba apprised of progress.
Roger (who will learn again to use Github).

**Comment by LaurencePeanuts** [comment]
Yes I'll start working on Stage 1 now. Thanks!

**Comment by LaurencePeanuts** [comment]
Done with Stage 1!

**Comment by rogerblandford** [comment]
Great
When can I see it?
Roger

**Comment by drphilmarshall** [comment]
Just saw Laurence: she pointed out I need to add in your new parts, Roger - I can't see them in Mac Preview, so Im installing Acrobat Reader. We really need to practice your GitHub useage :-) I'll fix Stage 2 and 3 in the next half hour.

**Comment by LaurencePeanuts** [comment]
ok i'm done adding the last figure, I've pushed one last time and I'm letting you edit, Phil.
I need to correct couple things on Fig 5 (it doesn't have abc yet) but I won't touch the tex file.

**Comment by rogerblandford** [comment]
OK Thanks for all the editing.
Roger

**Comment by rogerblandford** [comment]
Sorry.
I thought everyone used Adobe…
Roger

**Comment by drphilmarshall** [comment]
Thanks Laurence! I just finished with Stage 2 and 3, so I think we have everything done from `prodesRB.pdf`. Roger, I think the detailed approach and tree presentation parts of Stage 1 should stay where they are: lets keep Stage 2 to be the inclusion of local data, can we? There's plenty to do there and it is conceptually more separate. I put the sub heading back as well - hope this is OK.

The PDF is available to download from [this link](https://github.com/rogerblandford/Music/raw/master/doc/proposals/2016NASA/prodes.pdf), but it'd be better if you could just `git pull` and get the current version of the latex. What time's the deadline?

**Comment by rogerblandford** [comment]
"Thanks Laurence! I just finished with Stage 2 and 3, so I think we have everything done from prodesRB.pdf. Roger, I think the detailed approach and tree presentation parts of Stage 1 should stay where they are: lets keep Stage 2 to be the inclusion of local data, can we? There's plenty to do there and it is conceptually more separate. I put the sub heading back as well - hope this is OK."

I don’t feel strongly but I followed  the organizational rather than the scientific logic as this may matter most.
Stage 1 - almost done
Stage 2 - to be supported by this proposal
Stage 3 - where this is headed and not supported because it involves data not yet taken.

If we follow your scheme you will have to change some of the early writing.
